client/styles.css

```diff
@@ -31,6 +31,8 @@ label {
 	position: fixed;
 	top: 30px;
 	right: 30px;
+	overflow: scroll;
```

Comment:
You can set `overflow: auto;` instead, which would make the bar only appear when necessary.